### ISSUE TITLE
Use Display Name As File Name When Saving Torrent File

### ISSUE THREAD

**Issue by leoherzog** [aria2/aria2#1435]
When using `--bt-save-metadata`, aria2 creates a `.torrent` file with a filename of [the hex encoded info hash](https://aria2.github.io/manual/en/html/aria2c.html#cmdoption-bt-save-metadata). It would be cool if we could instead create files with pretty, human-readable names. Perhaps something like a `--bt-use-display-name-as-filename` flag, that would use [the `dn=` property of the magnet link](https://en.wikipedia.org/wiki/Magnet_URI_scheme#Design) if available?

**Comment by Jimmy-Z** [comment]
Yes, I'd want that too, I actually wrote a script to rename torrent files afterwards.

**Comment by alpha7504** [comment]
> Yes, I'd want that too, I actually wrote a script to rename torrent files afterwards. #1380

Hi, might be too late to ask but would you mind sharing the script if you still use it.

**Comment by Jimmy-Z** [comment]
I haven't used it in years but I dig it up: https://gist.github.com/Jimmy-Z/ac36346d646774e992617790435c5a94

No guarantee it still works though.

**Comment by alpha7504** [comment]
Thanks
